docs/tutorials/clickstream-docker.rst

```diff
@@ -7,135 +7,108 @@ These steps will guide you through how to setup your environment and run the cli
 
    -  `macOS <https://docs.docker.com/docker-for-mac/install/>`__
    -  `All platforms <https://docs.docker.com/engine/installation/>`__
 
 -  `Git <https://git-scm.com/downloads>`__
 
----------------------
-Download the Tutorial
----------------------
-
-Download and start the KSQL clickstream container. This container
-image is large and contains |cos|, Grafana, and Elasticsearch.
-Depending on your network speed, this may take up to 10-15 minutes.
-The ``-p`` flag will forward the Grafana dashboard to port 33000 on
-your local host.
-
-.. codewithvars:: bash
-
-    $ docker run -p 33000:3000 -it confluentinc/ksql-clickstream-demo:|release| bash
-
-Your output should resemble:
-
-.. codewithvars:: bash
-
-    Unable to find image 'confluentinc/ksql-clickstream-demo:|release|' locally
-    latest: Pulling from confluentinc/ksql-clickstream-demo
-    ad74af05f5a2: Already exists
-    d02e292e7b5e: Already exists
-    8de7f5c81ab0: Already exists
-    ed0b76dc2730: Already exists
-    cfc44fa8a002: Already exists
-    d9ece951ea0c: Pull complete
-    f26010779356: Pull complete
-    c9dad5440731: Pull complete
-    935591799d9d: Pull complete
-    696df0f65482: Pull complete
-    14fd98e52325: Pull complete
-    fcbeb94bace2: Pull complete
-    32cca4f1567d: Pull complete
-    5df0d25e7260: Pull complete
-    e16097edc4fc: Pull complete
-    72b33b348958: Pull complete
-    015da01a41b0: Pull complete
-    80e29f47abe0: Pull complete
-    Digest: sha256:f3b2b19668b851d1300f77aa8c2236a126b628b911578cc688c7e0de442c1cd3
-    Status: Downloaded newer image for confluentinc/ksql-clickstream-demo:latest
-    $
-
-You should now be in the Docker container and the remaining steps are run from within the container.
-
-----------------------------------------------
-Configure and Start Elastic, Grafana, and |cp|
-----------------------------------------------
-
-#.  Start  Elasticsearch.
 
-    .. code:: bash
-
-       $ /etc/init.d/elasticsearch start
+-----------------------------
+Download and run the tutorial
+-----------------------------
 
-    Your output should resemble:
+The tutorial is built using Docker Compose. It brings together several 
+Docker images with the required networking and dependencies. The images
+are quite large and depending on your network connection may take 
+10-15 minutes to download.
 
-    .. code:: bash
+#. Clone the Confluent KSQL repository.
 
-        [....] Starting Elasticsearch Server:sysctl: setting key "vm.max_map_count": Read-only file system
-        . ok
+   .. code:: bash
 
-#.  Start Grafana.
+       $ git clone https://github.com/confluentinc/ksql.git
+       $ cd ksql
 
-    .. code:: bash
+#. Switch to the correct |cp| release branch:
 
-        $ /etc/init.d/grafana-server start
+   .. codewithvars:: bash
+   
+       $ git checkout |release_post_branch|
 
-    Your output should resemble:
+#. Navigate to the KSQL repository ``docs/tutorials/`` directory and launch the tutorial in
```

Comment:
I think this should be "Navigate to the KSQL repository `ksql-clickstream-demo` directory..."